topics/templates/techpreview.adoc

```diff
@@ -1,14 +1,12 @@
-ifeval::[{project_product}==true]
 [NOTE]
 ====
 ifeval::[{tech_feature_disabled}!=false]
 {tech_feature_name} is *Technology Preview* and is not fully supported. This feature is disabled by default.
 
-To enable start the server with `-Dkeycloak.profile=preview`. For more details see
+To enable start the server with `-Dkeycloak.profile=preview` or `{tech_feature_setting}`. For more details see
```

Comment:
I assume techpreview.adoc is used elsewhere and those other places would not set tech_feature_setting currently